cmd/package.go

```diff
@@ -688,3 +687,13 @@ func constructBootCmdFromArguments(commandLine, customBoot, packageDir string) s
 	fmt.Println("Empty command line will be set for this image")
 	return ""
 }
+
+// AbsTarPathMatches tells whether the tar header name matches the path.
+// This function is needed since some tar files prefix its header names
+// with / and some not. NOTE: argument 'path' should start with '/' while
+// 'tarPath' should be header.Name.
+func AbsTarPathMatches(path, tarPath string) (res bool) {
```

Comment:
No need to be exported. Also, please, add a test so that we'll be sure about it when we are going to change it in the future.